CONTRIBUTING.md

```diff
@@ -79,6 +79,12 @@ There are also end to end tests to cover higher level user interactions, located
 
 ```bash
 cd ui-tests
+#install required packages for jlpm
+jlpm
+
+#install playwright
+npx playwright install
```

Comment:
Maybe we can stick to `jlpm` for consistency here?

```suggestion
jlpm playwright install
```

Reply:
Thanks @jtpio for pointing out.